packages/fx-core/src/common/deps-checker/constant/message.ts

```diff
@@ -4,51 +4,51 @@
 import { getDefaultString, getLocalizedString } from "../../localizeUtils";
 
 export const Messages = {
-  learnMoreButtonText: getLocalizedString("depChecker.learnMoreButtonText"),
-  defaultErrorMessage: [
+  // learnMoreButtonText: getLocalizedString("depChecker.learnMoreButtonText"),
```

Comment:
Why commentting this?

Reply:
There is no reference on this variable

Reply:
I see this variable is referenced in https://github.com/OfficeDev/TeamsFx/blob/dev/packages/fx-core/src/component/utils/depsChecker/bicepChecker.ts#L446.